docs/kb/semgrep-code/scan-engine-kill.md

```diff
@@ -135,10 +129,14 @@ to see both your RAM and your swap space usage in Linux.
 
 ## Parallelization
 
-Once you have determined the RAM sufficient to scan your large codebase, you can now introduce parallelization to speed up the subproccesses that can be parallelized. Larger Kubernetes pod and runners charge on a time basis, so it now becomes a study of feasiblity to achieve the optimal configuration.  
+Once you have determined the RAM sufficient to scan your large codebase, you can now introduce parallelization to speed up the scan.
+
+Per the previous section, you have determined the total memory required for a `-j 1` configuration. Now, you can begin testing different parallelization configurations, such as specifying `-j 2`, all the whilst still monitoring for any swap usage.
+
+If that succeeds, you can experiment with a `-j 4` configuration and continue monitoring scan time for improvements.  Keeping resource costs in mind, it serves well to determine the optimal configuration in bumping up the number of parallel jobs to speed up the scan. Thus, there is a balancing act between achieving faster scans and keeping your overhead costs down as much as possible. 
```

Comment:
```suggestion
If that succeeds, you can experiment with increasing `-j` and continue monitoring scan time for improvements. Finding the optimal balance depends on both resources used and scan time, since both can impact cost and efficiency.
```